src/Components/CoverImage/CoverImage.jsx

```diff
@@ -0,0 +1,22 @@
+import {useState } from "react"
+import style from "./CoverImage.module.css"
+export const CoverImagage=(url)=>{
+
+   const imageUrl=url.url[0].url
+
+const [isShown,setIsShown]=useState(false)
```

Comment:
It might be good to comment for line 7, as useState hasnt been covered yet this week, perhaps explaining why you chose to use it in this module?

Reply:
Okay. I will. I used it to check if a mouse entered an image and left it, and then I passed a class based on that.